doc/guides/cve_management_process.md

```diff
@@ -0,0 +1,137 @@
+# Node.js CVE management process
+
+The Node.js project acts as a [Common Vulnerabilities and Exposures (CVE)
+Numbering Authority (CNA)](https://cve.mitre.org/cve/cna.html).
+The current scope is for all actively developed versions of software
+developed under the Node.js project (ie.  https://github.com/nodejs).
+This means that the Node.js team reviews CVE requests and if appropriate
+assigns CVE numbers to vulnerabilities.  The scope currently **does not**
+include third party modules.
+
+More detailed information about the CNA program is available in
+[CNA_Rules_v1.1](https://cve.mitre.org/cve/cna/CNA_Rules_v1.1.pdf).
+
+# Contacts
+
+As part of the CNA program the Node.js team must provide a number
+of contact points.  Email aliases have been setup for these as follows:
+
+* **Public contact points**. Email address to which people will be directed
+  by Mitre when they are asked for a way to contact the Node.js team about
+  CVE-related issues. **cve-request@iojs.org**
```

Comment:
little nit: maybe address below as others ?

Reply:
@devnexen This PR literally moves the policy, I don't want to mix the move with updates to the policy.

The policy will need rework, though, since it isn't exactly what we do anymore, so I could address this later, but honestly, I'm not sure what "address below as others" means!

Reply:
I just meant the email address, doc style wise, nothing important though, it is ok as is :-)

Reply:
you think it should be "@nodejs.org"?

Reply:
I don't think we have forwards, so the @nodejs.org versions would not currently work.

Reply:
@sam-github sorry for being unclear :-) I meant
from
`CVE-related issues. **cve-request@iojs.org**`
to
`CVE-related issues. 
**cve-request@iojs.org**`